### PR TITLE
Blacklist wdat_wdt for Siemens IPC227G

### PR DESCRIPTION
**Pull request by rucoder**

### CODE REVIEW

File: pkg/grub/rootfs.cfg [hunk 1]
```diff
@@ -203,6 +203,12 @@ function set_x86_64_baremetal {
          set_global dom0_platform_tweaks "$dom0_platform_tweaks pci=realloc=off"
       fi
    fi
+   if [ "$smb_vendor" = "SIEMENS AG" ]; then
+      smbios -t 1 -s 5 --set smb_product
+      if [ "$smb_product" = "SIMATIC IPC227G" ]; then
+         set_global dom0_platform_tweaks "$hv_platform_tweaks modprobe.blacklist=wdat_wdt"
```
Comment (rene) [inline pkg/grub/rootfs.cfg:209]: `$hv_platform_tweaks modprobe.....` -> `$dom0_platform_tweaks modprobe....`

**Review by OhmSpectator — APPROVED** [review]
Looks fine. Don't we have other dependencies and it's ready to be merged?

**Comment by OhmSpectator** [comment]
@rucoder, could you please resolve the rebase conflicts?

**Comment by rucoder** [comment]
> @rucoder, could you please resolve the rebase conflicts?

rebased. There are no other dependencies for this PR

**Review by OhmSpectator — APPROVED** [review]

**Pull request merged**